Api/KanbanizeApiV1.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Chaplean\Bundle\KanbanizeApiClientBundle\Api;
+
+use Chaplean\Bundle\ApiClientBundle\Api\AbstractApi;
+use Chaplean\Bundle\ApiClientBundle\Api\Parameter;
+use Chaplean\Bundle\ApiClientBundle\Api\Route;
+use GuzzleHttp\ClientInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Class KanbanizeApiV1.
```

Comment:
Naming is not good. We need to have Api Suffix, so my proposal is KanbanizeLegacyApi and a comment to specify that it's V1